### DISCUSSION TITLE
Combining exported annotation files

### DISCUSSION THREAD

**Discussion by aszani** [tecoholic/ner-annotator#81] [General]
So, I want to merge all .json files I export from this tool. But I can't find the way. How could it be? Could you help me?

**Comment by alvi-khan** [comment]
Unfortunately, the software does not currently support this feature. It is being tracked in #41.

If you are comfortable doing so, it should be fairly trivial to write a script in Python that achieves this. I am providing one below to help you get started. Note that this script assumes that all of the .json files were created using the same list of classes.

```python
import json

files = ['1.json', '2.json']

classes = []
annotations = []

for file in files:
    with open(file, encoding='utf-8') as f:
        data = json.load(f)
        classes = data['classes']
        annotations = annotations + data['annotations']

data = {
    'classes': classes,
    'annotations': annotations
}

with open('out.json', 'w', encoding='utf-8') as f:
    json.dump(data, f)
```

**Reply by aszani** [reply]
thank you so much
